tests/Wallabag/ApiBundle/Controller/EntryRestControllerTest.php

```diff
@@ -497,6 +499,7 @@ public function testPostArchivedAndStarredEntry()
         $this->assertSame('http://www.lemonde.fr/idees/article/2016/02/08/preserver-la-liberte-d-expression-sur-les-reseaux-sociaux_4861503_3232.html', $content['url']);
         $this->assertSame(1, $content['is_archived']);
         $this->assertSame(1, $content['is_starred']);
+        $this->assertGreaterThanOrEqual($now->getTimestamp(), (new \DateTime($content['starred_at']))->getTimestamp());
```

Comment:
I have updated this test on my last push force sync (not a good idea but I would like to avoid too many commits). I don't know your rules concerning PR (when do we need to squash our commits, ...).

Reply:
It's ok.
Usually we let the user push commits and when everything is ok and if there are too much useless commits we ask to squash. Otherwise it's ok to have more than one commit and sometimes it's better when there are some reflexion between each commit.